recipe/meta.yaml

```diff
@@ -25,6 +25,8 @@ requirements:
     # Note that "testing" in conda-forge should be limited to testing
     # successful packaging, and not testing code.
     - compilers
+    - clang_osx-64 =11.1.0=hb91bd55_1  # [osx and x86_64]
+    - clang_osx-arm64 =11.1.0=h40be2b1_1  # [osx and arm64]
```

Comment:
Is there a reason you need to pin down to the build string? I'd be much better to only pin the version.

Reply:
Also I thought this was a libcxx issue? We marked the offending package as broken so this should not be needed.

Reply:
Hm.  I thought it was because we had had problems with a different build, but it appears that was for a different version.

Reply:
@erykoff do you recall what happened here?

Reply:
I don't know ... this could be related to the libcxx ... but I don't know and now it's moot anyway.

Reply:
It's moot for master, but not for the old versions that these builds are intended to be used with.